CHANGELOG.md

```diff
@@ -0,0 +1,349 @@
+## 1.29.0
+
+#### New Features
+
+* **Remote state files for Terraform inventory plugin**
+  The Terraform plugin for inventory configuration now supports both local and remote state files. (BOLT-1469)
+* **Reorganized command reference documentation**
+  The command reference documentation now shows a list of options available for each command, instead of having separate sections for commands and options. (BOLT-1422)
+
+#### Bug Fixes
```

Comment:
Maybe whatever script you are using is not picking up bug fixes as expected? I'm assuming that is why there is empty headers here and no notes for `1.29.1` which was bug fix only.